tests/test_routes.py

```diff
@@ -141,6 +141,337 @@ def test_cves_returns_422_for_non_existing_package_status(self):
         assert response.status_code == 422
         assert "Cannot find a status" in response.json["errors"]
 
+    def test_cve_updated_at_column(self):
+        """
+        Tests that updated_at is created automatically when a new CVE is added
+        and that it is changed on update.
+        """
+
+        cve_payload = payloads.cve1.copy()
```

Comment:
Could we add a comment here to make it clear that initially we're creating a CVE from scratch that doesn't exist? And then another lower down to explain that we're now updating it? I don't think that's necessarily obvious from the code.